storage/volume.go

```diff
@@ -5,6 +5,13 @@ package storage
 
 import "gopkg.in/juju/names.v2"
 
+type DeviceType string
+
+var (
+	DiskTypeLocal DeviceType = "local"
```

Comment:
This is mixing the overall "type DeviceType" with all prefixes that are "DiskType*" we probably want to be consistent here.